libs/cart/src/facades/cart/cart.facade.ts

```diff
@@ -4,19 +4,88 @@ import { Action, Store, select } from '@ngrx/store';
 
 import { DaffStoreFacade } from '@daffodil/core';
 import { DaffCart } from '../../models/cart';
-import * as fromCart from '../../reducers';
+import { DaffCartReducersState } from '../../reducers';
+import {
+  selectCartLoading,
+  selectCartValue,
+  selectCartErrors,
+  selectCartId,
+  selectCartSubtotal,
+  selectCartGrandTotal,
+  selectCartCoupons,
+  selectCartItems,
+  selectCartBillingAddress,
+  selectCartShippingAddress,
+  selectCartPayment,
+  selectCartTotals,
+  selectCartShippingInformation,
+  selectCartAvailableShippingMethods,
+  selectCartAvailablePaymentMethods,
+  selectItemErrors,
+  selectBillingAddressErrors,
+  selectShippingAddressErrors,
+  selectShippingInformationErrors,
+  selectShippingMethodsErrors,
+  selectPaymentErrors,
+  selectPaymentMethodsErrors
+} from '../../selectors';
+import { DaffCartErrors } from '../../reducers/cart-errors.type';
+import { DaffCartErrorType } from '../../reducers/cart-error-type.enum';
 
 @Injectable({
   providedIn: 'root'
 })
 export class DaffCartFacade implements DaffStoreFacade<Action> {
-
   loading$: Observable<boolean>;
   cart$: Observable<DaffCart>;
-  
-  constructor(private store: Store<fromCart.State>) {
-    this.loading$ = this.store.pipe(select(fromCart.selectCartLoadingState));
-    this.cart$ = this.store.pipe(select(fromCart.selectCartValueState));
+
+  cartErrors$: Observable<DaffCartErrors[DaffCartErrorType.Cart]>;
```

Comment:
I think we should keep the overall errors property too

Reply:
Changed in [`6128980`](https://github.com/graycoreio/daffodil/pull/681/commits/612898086944308222bcf7f16e9361fb4c2b4ebf).